sechub-integrationtest/src/main/java/com/daimler/sechub/integrationtest/api/FixedTestProject.java

```diff
@@ -0,0 +1,23 @@
+package com.daimler.sechub.integrationtest.api;
+
+import com.daimler.sechub.integrationtest.internal.TestScenario;
+
+public class FixedTestProject extends TestProject{
```

Comment:
I still have no idea what this class is suppose to do or used for. I would prefer a JavaDoc comment describing the purpose of the class.

Reply:
already done - see my former comment . I added javadoc for FixedTestProject and also for FixedTestuser

Reply:
done